lib/matplotlib/axes/_axes.py

```diff
@@ -1258,12 +1258,12 @@ def eventplot(self, positions, orientation='horizontal', lineoffsets=1,
             *orientation* parameter).
 
         orientation : {'horizontal', 'vertical'}, optional
-            Controls the direction of the event collections:
+            The direction of the event collections:
 
-                - 'horizontal' : the lines are arranged horizontally in rows,
-                  and are vertical.
-                - 'vertical' : the lines are arranged vertically in columns,
-                  and are horizontal.
+            - 'horizontal': the lines are arranged horizontally in rows,
```

Comment:
find "are vertical and arranged horizontally in rows" to be a little counterintutive

Reply:
I didn't change the wording, feel free to propose something better.

Reply:
Hmm, the problem is i don't even know what this wording is getting at, so I'll agree to table it for a different PR.